src/language/dafnyToolInstaller.ts

```diff
@@ -81,42 +81,49 @@ class DafnyToolInstaller {
               3.7.1.40621 Downloads: 754
               3.7.2.40713 Downloads: 324
    */
-  private static async getDotnetToolVersionUncached(context: ExtensionContext): Promise<string> {
+  private async getDotnetToolVersionUncached(context: ExtensionContext): Promise<string> {
```

Comment:
Changes in this file can be ignored since it's not used. Not sure why it's still in the codebase.